make/compiler/Makefile.gnu

```diff
@@ -118,6 +118,21 @@ ifeq ($(shell test $(GNU_GPP_MAJOR_VERSION) -lt 4; echo "$$?"),1)
 WARN_GEN_CFLAGS += -Wno-pointer-sign
 endif
 
+#
+# 2016/03/28: Help to protect the Chapel compiler from a partially
+# characterized GCC optimizer regression when the compiler is being
+# compiled with gcc 5.X.  Issue will be pursued after the release
```

Comment:
I would remove the "Issue will be pursued after the release" because it will quickly become stale and embarrassing if we don't get to it (I have a bunch of "I'll fix this soon" comments throughout the codebase, that I regret)

I would replace it with a link to the JIRA issue. e.g. 
  `See JIRA issue 194 for more info: https://chapel.atlassian.net/browse/CHAPEL-194`